jujugui/static/gui/src/app/components/button-dropdown/button-dropdown.js

```diff
@@ -1,219 +0,0 @@
-/* Copyright (C) 2017 Canonical Ltd. */
-
-'use strict';
-
-const classNames = require('classnames');
-const PropTypes = require('prop-types');
-const React = require('react');
-const ReactDOM = require('react-dom');
-const shapeup = require('shapeup');
-
-const SvgIcon = require('../svg-icon/svg-icon');
-const DropdownMenu = require('../dropdown-menu/dropdown-menu');
-
-require('./_button-dropdown.scss');
-
-/**
-  Creates a component that has an icon, which when clicked, opens a dropdown
-  menu with list items.
-*/
-class ButtonDropdown extends React.Component {
```

Comment:
All the components in the `components/shared` dir have been removed as we're now using the components from juju-react-components.